plugin/pkg/scheduler/algorithm/scheduler_interface.go

```diff
@@ -18,10 +18,24 @@ package algorithm
 
 import (
 	"k8s.io/kubernetes/pkg/api"
+	schedulerapi "k8s.io/kubernetes/plugin/pkg/scheduler/api"
 )
 
+// SchedulerExtender is an abstract interface for external processes to influence scheduling
```

Comment:
It would be helpful to mention the use-case of third-party resources influencing scheduling